src/EditorFeatures/CSharpTest/CodeActions/Preview/ErrorCases/ExceptionInCodeAction.cs

```diff
@@ -14,7 +14,7 @@ internal class ExceptionInCodeAction : CodeRefactoringProvider
     {
         public override Task ComputeRefactoringsAsync(CodeRefactoringContext context)
         {
-            context.RegisterRefactoring(new ExceptionCodeAction());
+            context.RegisterRefactoring(new ExceptionCodeAction(), context.Span);
```

Comment:
should this happen automatically for the user on their behalf?  jsut with an added bool passed along to say that we defaulted to this, instead of the client explicitly passing it?

Reply:
It's mostly in Tests & _a few_ refactorings where this happens (extract method, … (where it explicitly makes sense). And as discussed we users of this API to be mindful about what they pass in. And thus I'd advice against implicitly passing a value that is essentially fallback (in most situations).